### PR TITLE
spacing-and-style-tweaks-jc

### PR DESCRIPTION
**Pull request by joeclayallday**
**Various Style Tweaks**
• I just changed classNames on different components to adjust spacing
• Made Stacks list links match existing pattern

**All Top Level pages labels are aligned:**
Screen Shot 2023-06-26 at 1 02 37 PM

**More spacing between styles page elements:**
Screen Shot 2023-06-26 at 12 58 59 PM

FTUX background image is full bleed now:
Screen Shot 2023-06-26 at 7 25 19 PM

### CODE REVIEW

File: src/ui/pages/styles.tsx [hunk 1]
```diff
@@ -254,6 +254,7 @@ const Colors = () => (
     <h1 id="colors" className={tokens.type.h1}>
       Colors
     </h1>
+    <br />
```
Comment (neurosnap) [inline src/ui/pages/styles.tsx:257]: In general I would recommend not using `<br />` and instead opt for using margins.

**Review by neurosnap — CHANGES_REQUESTED** [review]
Tests are failing, probably just need to run `yarn fmt`

File: src/ui/pages/styles.tsx (hunk 1 above)
Comment (joeclayallday) [reply src/ui/pages/styles.tsx:257]: Sounds good!

Comment (joeclayallday) [reply src/ui/pages/styles.tsx]: Done, i removed all the HRs from the styles page and used `space-X`classNames instead, works like a charm

**Comment by joeclayallday** [comment]
All the table counts are spaced evenly with search boxes aligned left:
Screen Shot 2023-06-27 at 11 04 45 AM
Screen Shot 2023-06-27 at 11 04 48 AM
Screen Shot 2023-06-27 at 11 04 51 AM
Screen Shot 2023-06-27 at 11 05 08 AM
Screen Shot 2023-06-27 at 11 04 27 AM
Screen Shot 2023-06-27 at 11 04 33 AM
Screen Shot 2023-06-27 at 11 04 42 AM

File: src/ui/layouts/hero-bg-layout.tsx [hunk 2]
```diff
@@ -15,6 +15,9 @@ export const HeroBgLayout = ({
       className="flex flex-col flex-1 h-full bg-no-repeat bg-center bg-cover"
       style={{
         backgroundImage: "url(/background-pattern-v2.png)",
+        marginLeft: "-2rem",
```
Comment (madhuravius) [inline src/ui/layouts/hero-bg-layout.tsx:18]: we probably shouldn't do this here, can chat w/ you about what you wanna do (there's tailwind classes for this for example) OR we can toggle this somewhere else

Comment (joeclayallday) [reply src/ui/layouts/hero-bg-layout.tsx:18]: It works! Thanks!

**Review by madhuravius — APPROVED** [review]

**Pull request merged**